pwa/src/templates/organizationDetail/OrganizationDetailTemplate.tsx

```diff
@@ -36,127 +37,285 @@ export const OrganizationDetailTemplate: React.FC<OrganizationDetailTemplateProp
-          <img className={styles.logo} src={organizationLogo} alt="Organization logo" />
-          <div className={styles.tagsContainer}>
-            <ToolTip tooltip="GitHub">
-              <Tag label={t("GitHub")} icon={<GitHubLogo />} onClick={() => open("github")} />
-            </ToolTip>
-            <ToolTip tooltip="GitLab">
-              <Tag label={t("GitLab")} icon={<GitLabLogo />} onClick={() => open("gitlab")} />
-            </ToolTip>
-            <ToolTip tooltip="Website">
-              <Tag
-                label="www.rotterdam.nl"
-                icon={<FontAwesomeIcon icon={faGlobe} />}
-                onClick={() => open("https://rotterdam.nl")}
-              />
-            </ToolTip>
-            <ToolTip tooltip="Telefoonnummer">
-              <Tag
-                label="+31 - 102 - 671 - 625"
-                icon={<FontAwesomeIcon icon={faPhone} />}
-                onClick={() => navigate("tel:+31102671625")}
-              />
-            </ToolTip>
-            <ToolTip tooltip="EmailAddress">
-              <Tag
-                label="componenten-support@rotterdam.nl"
-                icon={<FontAwesomeIcon icon={faEnvelope} />}
-                onClick={() => navigate("mailto:componenten-support@rotterdam.nl")}
-              />
-            </ToolTip>
-          </div>
-        </div>
-      </div>
-
-      <Divider />
-
-      <div className={styles.section}>
-        <Heading2>Componenten</Heading2>
-        <TabContext value={currentTab.toString()}>
-          <Tabs
-            value={currentTab}
-            onChange={(_, newValue: number) => {
-              setCurrentTab(newValue);
-            }}
-            variant="scrollable"
-          >
-            <Tab
-              className={styles.tab}
-              label={
-                <BadgeCounter layoutClassName={styles.tabAmountBadge} number={_.toString(TempComponentsOwned.length)}>
-                  Eigen componenten
-                </BadgeCounter>
-              }
-              value={0}
-            />
-            <Tab
-              className={styles.tab}
-              label={
-                <BadgeCounter layoutClassName={styles.tabAmountBadge} number={_.toString(TempComponentsSupported.length)}>
-                  Ondersteunde componenten
-                </BadgeCounter>
-              }
-              value={1}
-            />
-            <Tab
-              className={styles.tab}
-              label={
-                <BadgeCounter layoutClassName={styles.tabAmountBadge} number={_.toString(TempComponentsUsed.length)}>
-                  Gebruikte componenten
-                </BadgeCounter>
-              }
-              value={2}
-            />
-          </Tabs>
-
-          <TabPanel className={styles.tabPanel} value="0">
-            <div className={styles.components}>
-              <ComponentCardsAccordionTemplate components={TempComponentsOwned} />
+      {tempOrganization && (
```

Comment:
Right now, we're duplicating the entire template based on wether or not the `tempOrganization` is set. If not, we're hardcoding the entire thing, if it is then we're using the dynamic one.

A better way to do this, is to make sure we have an organization set at all times. Right now, we're setting the `tempOrganization` like so:

`const tempOrganization = TEMPORARY_ORGANIZATIONS.find((organization) => organization.id === organizationId);`

Instead of thereafter directly returning the template (twice, depending on the state of `tempOrganization`, we could do the following:

```
if (!tempOrganization) {
 tempOrganization = TEMPORARY_ORGANIZATIONS[1]; // this is the Rotterdam one I think
}
```

Afterwards, the return statement of this component will **always** have an organization, so no need to check again _and_ no need to duplicate the template.

Note: I see that the current hardcoded part of this template has more data / layout (e.g. it wraps stuff in a `QuoteWrapper`, which wont happen with the test data. I'd still opt for above solution, if need be: update the temporary test data to mimic the intended layout.